robotcontainer.py

```diff
@@ -87,6 +106,7 @@ def __init__(self) -> None:
                 self.operatorInterface.chassisControls.rotationY,
             )
         )
+        self.elevator.setDefaultCommand(ElevatorIntakePosition(self.elevator))
```

Comment:
is the intaking position just generally idle?

Reply:
yes